CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Changelog
 
+## [1.28.0] - 2024-10-07
+
+## What's Changed
+
+- Replace remaining `winapi` usage with `windows-sys` by @kennykerr in https://github.com/rust-lang/rustup/pull/3802
+- Make `rustup show` output info in a more logical order by @majaha in https://github.com/rust-lang/rustup/pull/3225
+
+- Stop showing ETA after download is complete by @djc in https://github.com/rust-lang/rustup/pull/3827
+- consistently add context with file path when parsing fails by @Skgland in https://github.com/rust-lang/rustup/pull/3853
+- feat: support for parsing beta versions with tags in the toolchain by @roife in https://github.com/rust-lang/rustup/pull/3858
+- feat(cli): add `--quiet` to `rustup toolchain list` and friends by @rami3l in https://github.com/rust-lang/rustup/pull/3862
+- Use serde to encode/decode various TOML formats by @djc in https://github.com/rust-lang/rustup/pull/3864
+- Fix misleading "uninstalled toolchain" notification by @scimas in https://github.com/rust-lang/rustup/pull/3869
+- fix(dist/linux): handle the possible unavailability of `/proc` in `rustup-init.sh` by @rami3l in https://github.com/rust-lang/rustup/pull/3800
+- feat: add hint to run `rustup self` when err desc is `self` by @Xerxes-2 in https://github.com/rust-lang/rustup/pull/3901
+- implements quiet flag in `rustup-init.sh` by @vic1707 in https://github.com/rust-lang/rustup/pull/3910
+- feat(rustup-init): set log level to `WARN` on `-q` if `RUSTUP_LOG` is unset by @rami3l in https://github.com/rust-lang/rustup/pull/3911
+- Add loongarch64-unknown-linux-musl support by @heiher in https://github.com/rust-lang/rustup/pull/3921
+- feat(cli): warn when removing the default/active toolchain by @rami3l in https://github.com/rust-lang/rustup/pull/3924
+- feat(dist): refine suggestions regarding manifest checksum mismatches by @rami3l in https://github.com/rust-lang/rustup/pull/3923
+- Fix home_dir() and current_dir() regression by @djc in https://github.com/rust-lang/rustup/pull/3938
+- Add help message for missing toolchain by @jtr860830 in https://github.com/rust-lang/rustup/pull/3939
+- fix(rustup-mode): return `ExitCode(1)` when `update()` fails by @rami3l in https://github.com/rust-lang/rustup/pull/3952
+- fix(manifest): consider possible renames in `Component::try_new()` by @rami3l in https://github.com/rust-lang/rustup/pull/3991
+- Allow `rustup doc` to search for unions by @taylordotfish in https://github.com/rust-lang/rustup/pull/4004
+- feat(cli)!: set log level to `INFO`/`DEBUG` on `--quiet`/`--verbose` if `RUSTUP_LOG` is unset by @rami3l in https://github.com/rust-lang/rustup/pull/3987
+- Replace `winreg` with `windows-registry` by @InfyniteHeap in https://github.com/rust-lang/rustup/pull/3896
+
+## New Contributors
+
+- @kennykerr made their first contribution in https://github.com/rust-lang/rustup/pull/3802
```

Comment:
I think I'd prefer to skip this section, IMO it's mostly irrelevant to the CHANGELOG audience.